README.md

```diff
@@ -358,7 +358,7 @@ Using Docker recommendation pipeline
 docker run --rm -it \
   -v <absolute_path_to_folder_with_classes>:/in \
   -v <absolute_path_to_out_dir>:/out \
-  cqfn/aibolit-image
+  yegor256/aibolit-image
```

Comment:
@yegor256 not necessarily related to this PR, but the image was last updated 5 years ago.